src/lib/seam/connect/request-validation-error.ts

```diff
@@ -0,0 +1,19 @@
+import { SeamHttpApiError } from './api-error.js'
+
+// UPSTREAM: Should be provided by @seamapi/types.
+interface InvalidInputError {
+  type: 'invalid_input'
+  message: string
+  data?: unknown
+}
+
+export class SeamHttpInvalidInputError extends SeamHttpApiError {
+  override code: 'invalid_input'
+
```

Comment:
Eventually would like to add a property with type ZodError. Correct naming unclear at the moment since the API returns validation_errors which is a formatted Zod error.